profilepage/script/script.js

```diff
@@ -0,0 +1,158 @@
+var urls = ["https://fsd1.herokuapp.com/users/1/details",
+"https://fsd1.herokuapp.com/users/1/tweets",
+"https://fsd1.herokuapp.com/users/1/followers/suggestions",
+"https://fsd1.herokuapp.com/users/1/following"];
+
+var consolidate = urls.map(url => fetch(url).then(res => res.json()));
+Promise.all(consolidate).then(data => acceptURLs.call(data));//.then(res => console.log(res));
```

Comment:
Always make it a point to add `.catch` so that you handle the errors that might arise.